### PR TITLE
- Added getters for Model, VBMManager, VariableList

### PR DESCRIPTION
**Pull request by evsiretanu**
- Added a new Variable class
- Refactored some things

### CODE REVIEW

File: py/occam_base/model.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+    DEFAULT = 'default'
 
 
 class Model:
     """
     Wrapper class for Model
     """
 
-    def __init__(self, ref=None):
+    def __init__(self, ref=None) -> None:
```
Comment (tim-coutinho) [inline py/occam_base/model.py:15]: Does python2 support typing?

File: py/occam_base/model.py [hunk 2]
```diff
@@ -15,24 +21,30 @@ def __init__(self, ref=None):
...
 
     @fit_table.deleter
     def fit_table(self):
-        self.ref.deleteFitTable()
+        self._ref.deleteFitTable()
 
     def set(self, **kwargs):
         pass
+
+    def get_print_name(self) -> str:
```
Comment (tim-coutinho) [inline py/occam_base/model.py]: Could we use @properties here as well?

File: py/occam_base/model.py (hunk 1 above)
Comment (evsiretanu) [reply py/occam_base/model.py:15]: It doesn't. But everything from occam_base will only be used with python 3 versions of fit, sbsearch, etc

File: py/occam_base/model.py (hunk 2 above)
Comment (evsiretanu) [reply py/occam_base/model.py]: I left it out without property for the reasons we discussed before, regarding it crashing when the object is null and no checks are performed on c++ side.

Comment (evsiretanu) [reply py/occam_base/model.py]: I'll modify the pyocam.cpp to fix that and re commit

**Review by IanMcBride1 — APPROVED** [review]

**Pull request closed without merging**